package.json

```
@@ -1,13 +1,25 @@
 {
-  "name": "EngageSphere-Test-Design-Masterclass-turma-2",
+  "name": "engagesphere-test-design-masterclass",
   "version": "1.0.0",
   "description": "Sample project with a Node.js backend and a React frontend.",
   "scripts": {
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "lint": "eslint . --ignore-pattern 'backend/*' --ignore-pattern 'frontend/*'",
+    "api:run:cloud": "cypress run --record --tag 'API' --spec cypress/e2e/api-tests/**.cy.js",
+    "api:run": "cypress run --spec cypress/e2e/api-tests/**.cy.js",
+    "gui:run:cloud": "cypress run --record --tag 'GUI' --spec cypress/e2e/gui-tests/**.cy.js",
+    "gui:run": "cypress run --spec cypress/e2e/gui-tests/**.cy.js",
+    "component:run:cloud": "cypress run --record  --tag 'component' --component",
+    "component:run": "cypress run --component",
+    "accessibility:run": "cypress run --record --tag 'a11y' --spec cypress/e2e/accessibility-tests/**.cy.js",
+    "cloud:run": "npx cypress run && npm run --component"  
```

```suggestion
    "cloud:run": "cypress run && npm run --component"  
```

O comando `npx` é desnecessário em scripts do arquiv `pacakge.json`.